apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Mutation/SettlementReportMutation.cs

```diff
@@ -63,4 +69,12 @@ public async Task<bool> CancelSettlementReportAsync(
 
         return true;
     }
+
+    private bool IsPeriodAcrossMonths(Interval interval)
```

Comment:
Also might be a good idea to write a unit test for this one, with a few different cases. Method should also be static.